src/components/Footer.js

```diff
@@ -5,24 +5,26 @@ import Faq from "../icons/Faq";
 import Medicine from "../icons/Medicine";
 import House from "../icons/House";
 
-const FooterPage = styled.div`
+const FooterWrapper = styled.div`
   display: flex;
   width: 100vw;
   height: 52px;
   background-color: #003459;
   border: solid 1px;
   justify-content: space-between;
+  position: absolute;
+  bottom: 0;
```

Comment:
does it work like that? 
Use 0px instead;

Reply:
they both work the same. The value of 0 is 0 whether is px, %, rem, em etc....based on the documentation i found.